env0/resource_configuration_variable_test.go

```diff
@@ -602,4 +602,34 @@ resource "%s" "test" {
 			mock.EXPECT().ConfigurationVariableDelete(configVarImport.Id).Times(1).Return(nil)
 		})
 	})
+
+	t.Run("cant be empty value when isRequired and isReadOnly are true", func(t *testing.T) {
```

Comment:
add another case for update
where you update to empty value when `isRequired==isReadonly==true`

Reply:
f3606c7ad0fdc72a78fc51bee819696706761ca8